src/modules/DoNothingNode.js

```diff
@@ -0,0 +1,23 @@
+/*
+ * Demo Module. Does nothing.
+ */
+module.exports = function DoNothing(options) {
```

Comment:
Yes, this was for testing purposes... This is a demo module which gives the output image as it is...